README.md

```diff
@@ -94,6 +94,8 @@ Note: docker registry must be [v2](https://docs.docker.com/registry/spec/api/).
  * `max_concurrent_uploads`: *Optional.* Maximum concurrent uploads.
 
    Limits the number of concurrent upload threads.
+  
+  * `experimental`: *Optional.* Use Dockerd experimental option
```

Comment:
Could this instead be implied by `buildkit: true`? Does enabling this on its own make any difference?